.stylintrc

```diff
@@ -0,0 +1,10 @@
+{
+  "colons": "never",
+  "groupOutputByFile": true,
+  "indentPref": 2,
+  "prefixVarsWithDollar": false,
+  "quotePref": "single",
+  "semicolons": "never",
+  "sortOrder": "alphabetical",
+  "valid": false
+}
```

Comment:
I'd also add 

```
zeroUnits: true, // prefer 0 over 0px
trailingWhitespace: true,
efficient: true,
```

Reply:
`efficient` is on by default, as is `trailingWhitespace`. `zeroUnits` is default `never` as well